content/brazilian-portuguese/android-tictactoe/answer-key.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Gabarito - Jogo da Velha para Android"
+date: 2021-03-13T14:17:07.42-07:00
+hidden: true
+weight: 14
+---
+
+### Quebra-cabeça 1
+```xml
+<resources>
+ <string name="app_name">Jogo da Velha</string>
+</resources>
+```
+
+### Quebra-cabeça 2
+```kotlin
+Toast.makeText(this, "Welcome to Tic-Tac-Toe", Toast.LENGTH_LONG).show()
```

Comment:
"Welcome to Tic-Tac-Toe" might need translation.

Reply:
done